docs/en/01-how-to-build/android.md

```diff
@@ -198,3 +129,22 @@ MMDeploy provides a recipe as shown below for building SDK with ncnn as inferenc
 
   make -j$(nproc) && make install
   ```
+
+Please check [cmake build option spec](cmake_option.md)
+
+#### Build Demo
```

Comment:
Demos can be built by enable `-DMMDEPLOY_BUILD_EXAMPLES`, which is already presented in chapter `Build SDK Demos`. So we don't need to add a new `Build Demo` chapter any more.